### ISSUE TITLE
voided/closed ap's still showing in reviewer dashboard

### ISSUE THREAD

**Issue by MLWilkins** [dhenry-KCI/FredCo-Post-Go-Live-#107]
in life mapping issue auto generated new review cycles on countless Planning applications.  We are going through the older ones that are obviously expired and doing a void log to close them out.  However, they are still showing on the reviewers dashboards. We need voided applications to not show up in dashboards.  Resulting every system generated review on all this applications will take too long.  Need this resolved soon because reviewers are reviewing bogus applications. 

![image](https://user-images.githubusercontent.com/50637067/71291269-18d96180-2340-11ea-96e5-17033a32bea3.png)

![image](https://user-images.githubusercontent.com/50637067/71291305-30b0e580-2340-11ea-8c08-18331ba3b740.png)

**Comment by ScottVanSooy** [comment]
Unfortunately Mike, we don't have access to change the way the To-Do lists work.  I think  the best solution would be to create an SQL script to remove or result the reviews you don't want to see.  Before I run it though,  I'd like to talk to Christian or Dan and make sure there aren't any unintended consequences.

**Comment by MLWilkins** [comment]
@ScottVanSooy  is there way to eliminate all reviews that were automatically generated by IPS on 12/7/19?  Something happened during data migration that cause a number of applications from H7 to have a new review cycle generated.  @AReed327

**Comment by MLWilkins** [comment]
Just spoke with Ashlye B. On the permits side applications that are voided have all reviews waived automatically, as well as any outstanding fees.  We would like this on the Planning side as well.

**Comment by AReed327** [comment]
Please give me an update on this. There are 12 pages of reviews that are over due as of today. I need to get this sorted out. I voided Planning project 9613 as it is expired and the reviews are still showing. @ScottVanSooy @dhenry-KCI

**Comment by MLWilkins** [comment]
Once Scott fixes the issue please get Infor or Juanita or someone to void all those applications that expired and generated reviews. Too many for us to do. If there are valid applications then Infor needs to get rid of any reviews auto populated by IPS on 12/7/19.

Sent from my iPhone

On Dec 27, 2019, at 7:47 AM, AReed327 <notifications@github.com> wrote:

﻿
[EXTERNAL EMAIL]


Please give me an update on this. There are 12 pages of reviews that are over due as of today. I need to get this sorted out. I voided Planning project 9613 as it is expired and the reviews are still showing. @ScottVanSooy<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fScottVanSooy&umid=943de3e1-1fd7-440b-8273-93630b796300&auth=6ae1395420aead26648ba75b8b6ffb687c7ff70a-53ba8ed37cfeff7fd1623eda0fbb25bead8b92c8> @dhenry-KCI<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fdhenry%2dKCI&umid=943de3e1-1fd7-440b-8273-93630b796300&auth=6ae1395420aead26648ba75b8b6ffb687c7ff70a-5ff7a71f3492da6d70af1810dbe278d442bc6edf>

**Comment by ScottVanSooy** [comment]
I will focus on the first thing, I expect it to take a couple hours.

**Comment by ScottVanSooy** [comment]
Please Test and let me know if anything is out of place.

**Comment by AReed327** [comment]
I am looking now. 12 pages down to barely 5. I should be able to adjust from here. I’ll let you know if anything appears off.

Ashley Reed
Development Review Technician II
Department of Development Review
Frederick County Government
30 North Market Street
Frederick, Maryland 21701
(301)-600-1139
AReed@FrederickCountyMD.gov<mailto:AReed@FrederickCountyMD.gov>

**Comment by ScottVanSooy** [comment]
This formula closes everything out.  PLAN_CloseApp_PAOC  It looks like it would work as designed as long as it's called.  It's a Condition on the COMPLETE milestone.  My hunch is that Voided apps aren't making it to the Complete Milestone.

This formula is what happens when it's Voided.  PLN_VOID_AAF  I'm not sure if this is setting the Milestone to Complete correctly.  I kind of need Dan to look at this.

**Comment by AReed327** [comment]
It is not working because I have had to NA reviews on applications that are VOIDED. I agree that Dan should look at this and make the appropriate corrections. @dhenry-KCI @ScottVanSooy

**Comment by MLWilkins** [comment]
example of above.
![image](https://user-images.githubusercontent.com/50637067/71685101-54732300-2d65-11ea-8822-165ac67c4ce1.png)

![image](https://user-images.githubusercontent.com/50637067/71685126-6228a880-2d65-11ea-8c04-ff5f24d00d35.png)

**Comment by megisys** [comment]
2 Reviews with a Void status and no result. This has been addressed.

**Comment by MLWilkins** [comment]
applications that have been voided still appear on dashboard.  Just voided 1138, refreshed my infor page, and it still appears on reviewers to do list..

![image](https://user-images.githubusercontent.com/50637067/71917168-7269ca80-314d-11ea-99f6-b89ba6eca484.png)

![image](https://user-images.githubusercontent.com/50637067/71917296-beb50a80-314d-11ea-9fa7-5849543da1cc.png)

**Comment by ScottVanSooy** [comment]
Void Log fixed.  Adding a void Log will now Waive all Reviews, Inspections and Fees and reviews will no longer show in the dashboard.

**Comment by MLWilkins** [comment]
1138, which was voided before the fix last night, still shows in dashboard.  However, I voided a new one this morning (1159) and it no longer shows on the dashboard.  Worried that the projects we have voided before last night are still on users dashboards.  Any way to check this?


![image](https://user-images.githubusercontent.com/50637067/71979981-0474e000-31ed-11ea-941a-2461e4aea5fd.png)

![image](https://user-images.githubusercontent.com/50637067/71980026-1b1b3700-31ed-11ea-8099-5a686bccfa19.png)


New voided application
![image](https://user-images.githubusercontent.com/50637067/71980191-7b11dd80-31ed-11ea-963c-9621304195d2.png)
